app/subscriber/src/features/filter-media/FilterMediaLanding.tsx

```diff
@@ -0,0 +1,146 @@
+import { useFilterOptions } from 'components/navbar';
+import { PageSection } from 'components/section';
+import React from 'react';
+import { useContent } from 'store/hooks';
+import { Col, ISourceModel, Row, Show } from 'tno-core';
+
+import { MediaFilterTypes } from './constants';
+import { FilterMedia } from './FilterMedia';
+import * as styled from './styled';
+import { alphabetArray } from './utils';
+
+/** The FilterMediaLanding Consists of the filter to narrow down the results of media, as well as the results. It is returned in a two column layout*/
+export const FilterMediaLanding: React.FC = () => {
+  const {
+    dailyPrint,
+    sources,
+    weeklyPrint,
+    cpWire,
+    talkRadio,
+    onlinePrint,
+    television,
+    newsRadio,
+  } = useFilterOptions();
+  const [
+    {
+      mediaType: { filter },
+    },
+    { findContentWithElasticsearch, storeMediaTypeFilter: storeFilter },
+  ] = useContent();
+  const [activeFilter, setActiveFilter] = React.useState<MediaFilterTypes>(
+    MediaFilterTypes.DAILY_PRINT,
+  );
+  const [activeLetter, setActiveLetter] = React.useState<string>('All');
+  const [narrowedOptions, setNarrowedOptions] = React.useState<ISourceModel[]>([]);
+  const [activeSource, setActiveSource] = React.useState<ISourceModel | null>(null);
+
+  const labels = Object.values(MediaFilterTypes);
+  const determineOptions = () => {
```

Comment:
@jdtoombs Take a look at the way I've refactored the `MediaSection` component.  I think you can do away with the enum and this `determineOptions` lookup.